docs/source/workflow_outputs.rst

```diff
@@ -769,7 +769,6 @@ End use categories (e.g., Heating, Cooling, Hot Water) report occupied spaces fo
 .. note::
 
   Headroom is calculated as the panel's total rated breaker spaces minus occupied breaker spaces.
-  A positive value indicates panel availability whereas a negative value indicates panel constraint.
```

Comment:
I feel like we can get rid of this note altogether. Isn't it identical to footnote 770 [here](https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#hpxml-electric-panels)?